addons/utm/security/ir.model.access.csv

```diff
@@ -1,11 +1,11 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_utm_campaign,access_utm_campaign,model_utm_campaign,,1,0,1,0
+access_utm_campaign,access_utm_campaign,model_utm_campaign,base.group_portal,1,0,1,0
 access_utm_campaign_user,access_utm_campaign_user,model_utm_campaign,base.group_user,1,1,1,0
 access_utm_campaign_system,utm.campaign.system,model_utm_campaign,base.group_system,1,1,1,1
-access_utm_medium,access_utm_medium,model_utm_medium,,1,0,1,0
+access_utm_medium,access_utm_medium,model_utm_medium,base.group_portal,1,0,1,0
```

Comment:
Are you sure this won't impact link management (aka: coming from url with utm bits inside it, and having code that checks if it exists, creates new one on the flu, ...) ? as well as website_links app ?

Reply:
@tde-banana-odoo this one required a sudo in website_crm to fix it, if there is other places, we can fix where it's used